conn.go

```diff
@@ -88,7 +92,7 @@ type Conn struct {
 	r       *bufio.Reader
 	timeout time.Duration
 
-	uniq  chan uint8
+	uniq  chan int
```

Comment:
on the wire its either an int8 or an int16, but incase they change it in the future I thought an int would fit best. Could potentially use a uint though as we should only ever send >= 0 stream ids